README.md

```diff
@@ -49,7 +49,7 @@ delete and re-create a set of institutions, one of which is the standard default
 
 ### Local or dev installation
 
-The EQUELLA you are testing must have some java VM options set in `manager/equellaserver-config.sh JAVA_OPTS`) to properly enable autotesting:
+The EQUELLA you are testing must have some java VM options set in the build configuration `manager/equellaserver-config.sh JAVA_OPTS`) to properly enable autotesting:
```

Comment:
Not really a 'build configuration'... Maybe re-work the original to:

... set in `JAVA_OPTS` in `manager/equellaserver-config.sh`.

:thinking: